oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/HistoryDeleteEsDAO.java

```diff
@@ -66,7 +66,7 @@ public void deleteHistory(Model model, String timeBucketColumnName, int ttl) thr
         for (String prepareDeleteIndex : prepareDeleteIndexes) {
             client.deleteByIndexName(prepareDeleteIndex);
         }
-        String latestIndex = TimeSeriesUtils.latestWriteIndexName(model);
+        String latestIndex = client.formatIndexName(TimeSeriesUtils.latestWriteIndexName(model));
         if (!leftIndices.contains(latestIndex)) {
             client.createIndex(latestIndex);
         }
```

Comment:
`client.createIndex` already calls `formatIndexName` internally, I think you may need to `formatIndexName` before checking `if (!leftIndices.contains(latestIndex))` and use the original one in `client.createIndex`, am I right?

Reply:
> `client.createIndex` already calls `formatIndexName` internally, I think you may need to `formatIndexName` before checking `if (!leftIndices.contains(latestIndex))` and use the original one in `client.createIndex`, am I right?

Otherwise, the namespace with be prepended twice, perhaps

Reply:
`latestIndex` variable should be used in `contains` check only, `createIndex` should use the unformatted name like the old codes.

Reply:
done